ckanext/ontario_theme/templates/internal/scheming/form_snippets/resource_schema.html

```diff
@@ -0,0 +1 @@
+{% import 'macros/form.html' as form %}
```

Comment:
You can remove this file altogether with the `"hidden_in_form"` preset (see comment above)